samples/saga/migration/Core_6/Server.New/Program.cs

```diff
@@ -0,0 +1,92 @@
+﻿//#define POST_MIGRATION
+
+using System;
+using System.Data.SqlClient;
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.Persistence.Sql;
+// ReSharper disable RedundantUsingDirective
+using NServiceBus.Features;
+using NServiceBus.Routing;
+using NServiceBus.Transport;
+// ReSharper restore RedundantUsingDirective
```

Comment:
ignore my comment :) Although it's kinda weird that our samples contain resharper specific comments but I guess that's a different topic.